assets/js/spec/core_test.js

```diff
@@ -0,0 +1,45 @@
+describe('App', function() {
+    beforeEach(function() { jasmine.Ajax.install(); });
+    afterEach(function() { jasmine.Ajax.uninstall(); });
+
+    it('builds request correctly for /new', function() {
+        var fixture =
+            '<input type="number" name="size" value="4">' +
+            '<select name="vs"><option value="minimax"></option></select>';
+        setFixtures(fixture);
+        expect(getNewGameUri()).toBe('new?size=4&vs=minimax');
+    });
+
+    it('creates buttons based on JSON response', function() {
+        testButtonSetup(3);
+        expect($('.move').length).toBe(9);
```

Comment:
It is usually preferred to keep classes for just styling and use data attributes for grabbing elements with js

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes